static/css/taxbrain.css

```diff
@@ -8440,6 +8440,18 @@ label {
   border-right: 2px solid #dddddd;
   padding-left: 30px;
 }
+.table-check-all {
```

Comment:
I can move bootstrap to `package.json` if that will help simplify things up. That way we won't have to do `bower install` in addition to `npm install`